### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to 1.2.22
- Fix plugins link page
- Upgrade to PHP8

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by yalh76** [comment]
> 😜 [Test Badge](https://ci-apps-dev.yunohost.org/ci/job/1412) [](https://ci-apps-dev.yunohost.org/ci/job/1412)

Permissions should be fixed

File: scripts/_common.sh [hunk 1]
```diff
@@ -4,9 +4,9 @@
 # COMMON VARIABLES
 #=================================================
 
-YNH_PHP_VERSION="7.3"
+YNH_PHP_VERSION="8.0"
 
-pkg_dependencies="php${YNH_PHP_VERSION}-cli php${YNH_PHP_VERSION}-opcache php${YNH_PHP_VERSION}-curl php${YNH_PHP_VERSION}-json php${YNH_PHP_VERSION}-gd php${YNH_PHP_VERSION}-zip php${YNH_PHP_VERSION}-xml php${YNH_PHP_VERSION}-mbstring php${YNH_PHP_VERSION}-mysql"
```
Comment (yalh76) [inline scripts/_common.sh:9]: seems that `php${YNH_PHP_VERSION}-json` is required

File: scripts/install [hunk 2]
```diff
@@ -21,16 +26,17 @@ ynh_abort_if_errors
 
 domain=$YNH_APP_ARG_DOMAIN
 path_url=$YNH_APP_ARG_PATH
-admin=$YNH_APP_ARG_ADMIN
 is_public=$YNH_APP_ARG_IS_PUBLIC
+admin=$YNH_APP_ARG_ADMIN
 email=$(ynh_user_get_info --username=$admin --key=mail)
+phpversion=$YNH_PHP_VERSION
```
Comment (yalh76) [inline scripts/install:32]: Why adding that ?

File: scripts/upgrade [hunk 3]
```diff
@@ -18,13 +18,10 @@ app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get --app=$app --key=domain)
 path_url=$(ynh_app_setting_get --app=$app --key=path)
-admin=$(ynh_app_setting_get --app=$app --key=adminusername)
-is_public=$(ynh_app_setting_get --app=$app --key=is_public)
+admin=$(ynh_app_setting_get --app=$app --key=admin)
```
Comment (yalh76) [inline scripts/upgrade:21]: new key `admin` should be managed during upgrade from previous versions and key `adminusername` should be removed if you choose to switch from `adminusername` to `admin`

File: scripts/upgrade [hunk 4]
```diff
@@ -18,13 +18,10 @@ app=$YNH_APP_INSTANCE_NAME
...
 final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 db_name=$(ynh_app_setting_get --app=$app --key=db_name)
-db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
-phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
-email=$(ynh_user_get_info --username=$admin --key=mail)
+phpversion=$YNH_PHP_VERSION
```
Comment (yalh76) [inline scripts/upgrade:24]: This should not be needed here

**Review by yalh76 — CHANGES_REQUESTED** [review]
It's better to do your commits in a dedicated branch instead of directly into testing branch, to let other maintainer also do their own PR and only doing merge request on testing when your PR is ready

File: scripts/_common.sh (hunk 1 above)
Comment (ericgaspar) [reply scripts/_common.sh:9]: PHP8.0 as json as default

File: scripts/install (hunk 2 above)
Comment (ericgaspar) [reply scripts/install:32]: Because I had issues without it. (the problem could come from PHP helper)

**Comment by yalh76** [comment]
> > It's better to do your commits in a dedicated branch instead of directly into testing branch, to let other maintainer also do their own PR and only doing merge request on testing when your PR is ready
> 
> the dedicated branch was #135, merged into testing

sorry then, I haven't seen that PR

File: scripts/install (hunk 2 above)
Comment (yalh76) [reply scripts/install:32]: I thunk the solution is just to add `phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)` after `ynh_add_fpm_config` or having this helper export `phpversion`

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade (hunk 4 above)
Comment (ericgaspar) [reply scripts/upgrade:24]: Removing this leads to an error: https://ci-apps-dev.yunohost.org/ci/job/2143

File: scripts/upgrade (hunk 3 above)
Comment (ericgaspar) [reply scripts/upgrade]: I reverted this to as it was.

**Pull request merged**